src/gl/interface/gl-objects-buffers.ads

```diff
@@ -17,17 +18,19 @@ package GL.Objects.Buffers is
    type Buffer is new GL_Object with private;
 
    procedure Bind (Target : Buffer_Target; Object : Buffer'Class);
+   procedure Bind_Buffer_Base (Target : Buffer_Target; Index : UInt;
+                               Object : Buffer'Class);
 
    function Current_Object (Target : Buffer_Target) return Buffer'Class;
 
    generic
       with package Pointers is new Interfaces.C.Pointers (<>);
-   procedure Load_To_Buffer (Target : Buffer_Target;
+   procedure Load_To_Buffer (Target : Buffer_Target'Class;
                              Data   : Pointers.Element_Array;
                              Usage  : Buffer_Usage);
 
    -- Use this instead of Load_To_Buffer when you don't want to copy any data
-   procedure Allocate (Target : Buffer_Target; Number_Of_Bytes : Long;
+   procedure Allocate (Target : Buffer_Target'Class; Number_Of_Bytes : Long;
                        Usage  : Buffer_Usage);
```

Comment:
This change is unnecessary since Allocate is not a generic procedure and therefore will be inherited for `Texture_Buffer_Target`. Instead, add `'Class` to the generic subroutines below (`Map`, `Pointer`, `Set_Sub_Data`), they need it.

Reply:
OK
Hopefully correct this time.